start/sw/mobile-app/mobile-devassistant.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Google Android Studio
+page: mobile-app
+order: 5
+---
+
+# How to install Google Android Studio on Fedora
+
+
+First of all install [DevAssistant](https://devassistant.org) project.
```

Comment:
I would create the DevAssistant installation guide in DevAssistant content and just make a link there, because otherwise including it in every section is pretty redundant and harder to maintain.